src/main/java/duke/task/ToDos.java

```diff
@@ -0,0 +1,40 @@
+package duke.task;
+
+import duke.DukeException;
+
+/**
+ * Class that represents todo task.
+ */
+public class ToDos extends Task {
```

Comment:
I think the class name could be "ToDo" or "ToDoTask" instead of "ToDos" as the "s" at the end may be a bit misleading as it may sound like it is referring to a list of todo tasks.